examples/detection_example.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include "../include/feature_lib.h"
+
+std::vector<uint8_t> matToVector(const cv::Mat &mat) {
+  std::vector<uint8_t> result;
+  cv::Mat cont = mat.isContinuous() ? mat : mat.clone();  // make sure the Mat is continuous
+  cont.reshape(1, 1).copyTo(result);  // copy from a 1D Mat
+  return result;
+}
+
+// Calculates keypoints and image descriptors
+// Takes an image file path as input
+// prints key points and image descriptors to stdout declared in associated header
```

Comment:
1. Start this sentence with a capital letter.
2. `to stdout declared in associated header` -- what? You need to remove the info about the header, I believe.

Reply:
my bad